### ISSUE TITLE
Missing STEREO EUVI data May 31, 2014 - June 2, 2014

### ISSUE THREAD

**Issue by takucera** [Helioviewer-Project/JHelioviewer-SWHV#140]
Hi, I am trying to make a movie including STEREO EUVI data from the end of May to the start of June 2014. The data from May 31, 2014 - June 2, 2014 is not coming up from either IAS or GSFC in JHelioviewer. It does seem to exist in the STEREO data base.

**Comment by ebuchlin** [comment]
Indeed, these data are not present in the GSFC JP2000 archive: https://helioviewer.org/jp2/EUVI-A/2014/06/
